archlinuxcn/mesa-git/PKGBUILD

```diff
@@ -91,7 +91,7 @@ package_vulkan-mesa-layers-git() {
   provides=('vulkan-mesa-layers')
   depends=('wayland' 'libxcb' 'python')
   conflicts=('vulkan-mesa-layer-git' 'vulkan-mesa-layers')
-  replaces=('vulkan-mesa-layer-git')
+  replaces=('vulkan-mesa-layer')
```

Comment:
You must not replace official package and VCS package must also not replace non-VCS packages in general.